docs/deployment/Azure/cloud_init_readme.md

```diff
@@ -32,11 +32,44 @@ First we need to setup the devbox that we use to operate on.
     ```
 4. [configure](cloud_init_configure.md) your azure cluster. Put `config.yaml` under `src/ClusterBootstrap`
 
-5. run batch script to deploy the cluster
+5. Run batch script to deploy the cluster
     ```
     ./deploy.sh
     ```
 
+6. Further steps required if you want to submit job to confirm that the deployment is successful. 
+
+To manually connect to the node, run:
+
+```./cluster_ctl.py connect <role> [id]```
+
+After finishing deploying the cluster, we need to further configure it. 
+Connect to infra node, and use following command to enter the mysql docker container:
+
+Login to infra node by `./cluster_ctl.py connect infra 0`, then on infra node,
+
+```docker exec -it $(sudo docker ps | grep mysql_mysql | awk '{print $1}') bash```
+
+to enter the mysql docker container.
+
+Enter the mysql container, run (-u -p according to your own setting in config):
+`mysql -u <user name> -<password>`
+
+Enter mysql, use `show databases` to list db, and run:
+`use <DLWS-initiated db>;`
+
+Add following entries for users:
```

Comment:
We should provide Admin commands to view and update ACL related fields in DB. But not in this PR.

Reply:
Theoretically, we only need to add to `acl` table. `user-synchronizer` will update/insert user info to `identity` table every hour.